src/docker.test.ts

```diff
@@ -12,6 +12,11 @@ jest.unstable_mockModule("./util.js", (): typeof import("./util.js") => ({
   execBashCommand: jest.fn<typeof import("./util.js").execBashCommand>(),
 }));
 
+const cache = jest.mocked(await import("@actions/cache"));
+const core = jest.mocked(await import("@actions/core"));
+const util = jest.mocked(await import("./util.js"));
```

Comment:
Question: what is the rationale behind the ordering of the imports here? I'm seeing that in the `describe` block in the previous version of the code, util is before docker.  Then the reverse is true in the `beforeAll` block.  I think we usually do alphabetical order, so I'm wondering if this is a mistake or if there is something I'm missing.

Reply:
Are you comparing different files? In this file, `util` consistently precedes `docker`. We import the unit under test last so that we don't mix it with the mocked dependencies. We import built-in node modules before third-party dependencies before our own dependencies and sort alphabetically within each section.

Reply:
Got it.  That makes sense! I don't know where I was seeing the inconsistency...my brain....hmmmmmmm.  Although I didn't notice the discrepancy between `util` needing to be mocked and `docker` simply needing to be imported.  Thanks for clearing this up!

Reply:
You were probably just comparing different files.